pkg/interceptors/cel/cel.go

```diff
@@ -136,21 +136,24 @@ func (w *Interceptor) ExecuteTrigger(request *http.Request) (*http.Response, err
 func evaluate(expr string, env *cel.Env, data map[string]interface{}) (ref.Val, error) {
 	parsed, issues := env.Parse(expr)
 	if issues != nil && issues.Err() != nil {
-		return nil, issues.Err()
+		return nil, fmt.Errorf("failed to parse expression %#v: %s", expr, issues.Err())
 	}
 
 	checked, issues := env.Check(parsed)
 	if issues != nil && issues.Err() != nil {
-		return nil, issues.Err()
+		return nil, fmt.Errorf("expression %#v check failed: %s", expr, issues.Err())
 	}
 
 	prg, err := env.Program(checked)
 	if err != nil {
-		return nil, err
+		return nil, fmt.Errorf("expression %#v failed to create a Program: %s", expr, err)
```

Comment:
v.minor: my first thought after this is what is a Program? Unless one is super familiar with CEL, this message doesn't seem as useful as the other errors here. But at the same time, I can't really think of anything better 🤷‍♂️

Reply:
Yeah, I had the same thoughts, it's really just a marker as to where it's failing, the previous errors weren't particularly granular.

The name isn't particularly significant, other than it's failing on _"line 149"_, which would be unusual, and so, somewhere to start looking.